File: Calculator.py

def calculate():
    num1 = float(input("Enter First Number: "))
    op = input("Enter operator: ")
    num2 = float(input("Enter Second Number:"))

    if op == "+":
        result = num1 + num2
    elif op == "-":
        result = num1 - num2
    elif op == "*":
        result = num1 * num2
    elif op == "/":
        result = num1 / num2
    else:
        result = "Invaild Operator"

    return result

try:
    print(calculate())

except ZeroDivisionError as err:
    print(err)

except ValueError as err2:
    print(err2)

File: Calculator.js
let num1 = Number(window.prompt("Enter first number: "))
let op = String(window.prompt("Enter the operator: "))
let num2 = Number(window.prompt("Enter second number: "))
let result

switch (op) {
    case  "+":
        result = num1 + num2
        break;

    case  "-":
        result = num1 - num2
        break;

    case  "*":
        result = num1 * num2
        break;

    case  "/":
        result = num1 / num2
        break;

    default:
        result = "Invaild Operator"
        break;
}

console.log(result);